src/components/NavBar.module.css

```diff
@@ -1,29 +1,33 @@
 :root {
   --main-color: #062d1d;
-  --secondary-color: #E3AA4D;
+  --secondary-color: #e3aa4d;
   --support-color: #ffffff;
 }
 
 .background {
   background-color: var(--main-color);
 }
 
-.navLink {  
-  color: var(--support-color)!important;
+.navLink {
+  color: var(--support-color) !important;
+  font-family: "Merriweather", serif;
 }
 
-.navLink:hover {  
+.navLink:hover {
   color: var(--secondary-color) !important;
+  font-family: "Merriweather", serif;
   font-size: 1.05rem;
   transition: 0.2s;
 }
```

Comment:
You don't need to add the font-family to the hover as you added it to the main class.

Reply:
done